exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hour: 0, minute: 0
+
+  @doc """
+  Returns a string representation of a clock:
+
+      iex> Clock.new(8, 9) |> to_string
+      "08:09"
+  """
+  @spec new(integer, integer) :: Clock
```

Comment:
Ahh, yeah, that's kind of what I was saying... @tejasbubane - want to take a swing at implementing at type for this module so these typespecs are doing what we expect them to do?